pkg/database/realm.go

```diff
@@ -571,6 +571,38 @@ func (r *Realm) FindAuthorizedApp(db *Database, id interface{}) (*AuthorizedApp,
 	return &app, nil
 }
 
+// ListMobileApps gets all the mobile apps for the realm.
+func (r *Realm) ListMobileApps(db *Database) ([]*MobileApp, error) {
+	var apps []*MobileApp
+	if err := db.db.
+		Unscoped().
```

Comment:
Why unscoped? that would include deleted.

Reply:
I went with the same approach of being able to resurrect apps (like you can resurrect keys). So, this is A-OK (the UI supports it, etc.)

![image](https://user-images.githubusercontent.com/3063869/94289564-e25ff100-ff26-11ea-8983-adecf7fdd1cc.png)